frontend/packages/knative-plugin/src/components/overview/__tests__/KnativeOverview.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as _ from 'lodash';
+import { PodRing } from '@console/shared';
+import { ResourceSummary } from '@console/internal/components/utils';
+import { KnativeOverview } from '../KnativeOverview';
+import { mockOverviewItemData } from './overview-knative-test-data';
+
+describe('KnativeOverview', () => {
+  it('should not render PodRing if obj.kind is not RevisionModel.kind', () => {
```

Comment:
This test name doesn't appear to match the test...

Reply:
fixed